package.json

```diff
@@ -1,10 +1,12 @@
 {
-  "name": "sage",
   "private": true,
   "workspaces": [
     "apps/*",
     "libs/*"
   ],
+  "engines": {
+    "node": ">= 16"
```

Comment:
Any reason we're starting with 16 instead of 18? 16 stopped receiving security support on 10-18-2022

Reply:
Yes, you are right. Does that mean that people should be forced to use 18 to develop within our ecosystem? If I recall, we are saying that we know that our code works with this minimum version of node.

Reply:
You're correct, but with that stance we have to have some sort of testing scenarios in place to verify 16 and 18 both work (probably from CI). If that's in the plans then I think this is totally reasonable, if not I'd say that we should probably put it on the docket.

Reply:
See the latest [commit](https://github.com/Kajabi/sage/pull/50/commits/a94f8d2ec4e3ddead98d190bdf831b054eabc509)